Makefile.am

```diff
@@ -1,4 +1,9 @@
 ACLOCAL_AMFLAGS = -I m4
 
 SUBDIRS = src man test
-dist_doc_DATA = README COPYING.txt GFDLv1.3.txt
+dist_doc_DATA = README COPYING.txt GFDLv1.3.txt ChangeLog.md
+EXTRA_DIST = generate-changelog.sh
+
+ChangeLog.md: generate-changelog.sh
```

Comment:
I suspect this line is forcing you to distribute generate-changelog.sh.
Remove the dependency on this script and you won't need it anymore.
I made the same mistake on libvecpf. :-(